README.md

```diff
@@ -99,6 +99,55 @@ documentation.
 
 # Usage
 
+### Influence Functions
+
+For influence computation, follow these steps:
+
+1. Wrap your model and loss in a `TorchTwiceDifferential` object
+2. Compute influence factors by providing training data and inversion method
+
+Using the conjugate gradient algorithm, this would look like:
+```python
+import torch
+from torch import nn
+from torch.utils.data import DataLoader, TensorDataset
+
+from pydvl.influence import TorchTwiceDifferentiable, compute_influences
+from pydvl.influence.inversion import InversionMethod
```

Comment:
I would add InversionMethod to `influence/__init__.py` for convenience while we are at it.

Reply:
Done